src/test/ui/cmse-nonsecure/cmse-nonsecure-call/wrong-abi-location-1.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0781]: The `"cmse-nonsecure-call"` ABI is only allowed on function pointers.
```

Comment:
Why did this not fail in CI? The message should now be lowercased.

Reply:
I have the feeling that the tests with:
```
// compile-flags: --target thumbv8m.main-none-eabi --crate-type lib
// only-thumbv8m.main-none-eabi
```
are ignored on CI 😕

Reply:
Oh, right. They are skipped by the PR builder, but the "real" CI should check them.